bool compliant
float64 goal_position


